File: eRecipes/UI/erecipes_desktop/lib/models/uplata.dart
import 'package:erecipes_desktop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'uplata.g.dart';

@JsonSerializable()
class Uplata{
  int? uplataId;
  double? iznos; 
  int? korisnikId;
  DateTime? datumUplate;
  Korisnik? korisnik;
  
   Uplata();
  
  factory Uplata.FromJson(Map<String,dynamic> json)=> _$UplataFromJson(json);

  Map<String,dynamic> toJson() => _$UplataToJson(this);
}

File: eRecipes/eRecipes.Model/Uplata.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class Uplata
    {
        public int UplataId { get; set; }
        public int KorisnikId { get; set; }
        public decimal Iznos { get; set; } = 10.00m;
        public DateTime DatumUplate { get; set; } = DateTime.Now;
        public Korisnik Korisnik { get; set; }
    }
}
